cdr_data_dictionary/generate_yaml.py

```diff
@@ -289,48 +282,14 @@ def _write_yaml_list(yaml, fields, value_list, index):
                 continue
             yaml.write('        ' + key + ':  ')
             sub_value = _process_value(value[0])
-            if sub_value:
+            if sub_value or isinstance(sub_value, bool):
                 _write_value(yaml, key, sub_value)
             else:
                 yaml.write('\n')
 
         yaml.write('\n')
 
 
-def _write_yaml_dict(yaml, fields, value_dict, index):
```

Comment:
It was never used as part of the workflow. so it was essentially "dead" code. it was unreachable. i felt like leaving it in would be confusing for new maintainers. it had been left in with the thought that it might be used later. but it isn't, so the risk seemed to outweigh the benefits.